rollup.config.js

```diff
@@ -1,22 +1,20 @@
-import babel from 'rollup-plugin-babel'
-import resolve from '@rollup/plugin-node-resolve'
-import commonjs from '@rollup/plugin-commonjs'
-import { terser } from 'rollup-plugin-terser'
```

Comment:
looks like you removed terser in this PR

Reply:
Good catch - merge conflict failure. Just updated this to reinstate terser